packages/effect/src/Number.ts

```diff
@@ -492,3 +492,26 @@ export const parse = (s: string): Option<number> => {
     ? option.none
     : option.some(n)
 }
+
+/**
+ * Returns the number rounded with the given precision.
+ *
+ * @param self - The number to round
+ * @param precision - The precision
+ *
+ * @example
+ * import { round } from "effect/Number"
+ *
+ * assert.deepStrictEqual(round(5,1234, 2), 5.12)
+ * assert.deepStrictEqual(nextPow2(17), 32)
```

Comment:
This can be removed.

Reply:
Sorry for that. I started to write the doc and stopped to write the tests first and forgot to come back to it.
There was also a mistake on the first line, by the way